frontend/src/queries/nodes/DataVisualization/Components/DisplayTab.tsx

```diff
@@ -23,6 +29,18 @@ export const DisplayTab = (): JSX.Element => {
                 />
             </div>
 
+            {isStackedBarChart && (
+                <div className="mt-1 mb-2">
```

Comment:
LemonCheckbox has a label field which I think will inline the components and look more complete